app/src/main/res/layout/state_fragment.xml

```diff
@@ -42,6 +82,7 @@
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent"/>
+      app:layout_constraintTop_toTopOf="parent"
+      app:layout_constraintVertical_bias="0.935" />
```

Comment:
This 0.935 seems like a magic number--where'd it come from?

Reply:
Changed approach

Reply:
removed